Source/IGListAdapter.m

```diff
@@ -552,22 +552,22 @@ - (BOOL)itemCountIsZero {
     return isZero;
 }
 
-- (IGListSectionMap *)sectionMapAdjustForUpdateBlock:(BOOL)adjustForUpdateBlock {
+- (IGListSectionMap *)sectionMapAdjustForUpdateBlock:(BOOL)usePreviousSection {
```

Comment:
Would `- (IGListSectionMap *)sectionMapUsingPreviousIfInUpdateBlock:(BOOL)usePreviousMapIfInUpdateBlock` or something be clearer?